contracts/DrawCalculatorV3.sol

```diff
@@ -88,14 +88,20 @@ contract DrawCalculatorV3 is IDrawCalculatorV3, Manageable {
         address _user,
         uint32[] calldata _drawIds,
         bytes calldata _pickIndicesForDraws
-    ) external view override returns (uint256[] memory prizesAwardable, bytes memory prizeCounts) {
+    ) external view override returns (
+        uint256[] memory prizesAwardable,
+        bytes memory prizeCounts,
+        uint64[][] memory drawPickIndices
```

Comment:
Haha this is funny!

The `_pickIndicesForDraws` are being passed in, decoded, then returned.  Nothing else.  This is like an elaborate ABI decoder.

The PrizeDistributor has no knowledge of the "picks".  Originally that's because we wanted the DC to be swappable.  

Returning the picks to the prize distributor means it's now aware of "picks", which completely negates the whole abstraction.

Reply:
Fixed in: https://github.com/pooltogether/v4-core/pull/274/commits/0f242fbb5b67045d4e9b4065f42a93bf239b439a